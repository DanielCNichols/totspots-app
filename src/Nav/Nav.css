nav {
  padding: 1% 0;
  color: black;
  display: inline-block;
}

nav a {
  color: white;
}

nav a:visited {
  color: white;
}

nav {
  height: 70px;
  position: absolute;
  padding: 0;
  width: 100%;
  top: 0;
  /* z-index: 10; */
  display: flex;
  align-items: center;
}

nav ul {
  display: flex;
  width: 100%;
  justify-content: space-evenly;
  padding: 0;
}

nav li {
  list-style: none;
  display: inline-block;
}

@media screen and (min-width: 800px) {
  nav ul {
    width: 30%;
    margin-left: auto;
  }

  nav {
    height: 50px;
    padding: 0;
    width: 100%;
    top: 0;
    z-index: 10;
    display: flex;
    align-items: center;
  }
}
