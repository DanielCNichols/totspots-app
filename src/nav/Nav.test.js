import React from 'react';
import ReactDOM from 'react-dom'
import Nav from './Nav.js'
import {MemoryRouter} from 'react-router-dom'


it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
  <MemoryRouter>
    <Nav/>
  </MemoryRouter>
  , div);
  ReactDOM.unmountComponentAtNode(div)
})