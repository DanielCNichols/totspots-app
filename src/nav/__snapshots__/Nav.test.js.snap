// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Nav as expected 1`] = `
<nav>
  <ul>
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/addVenue"
        onClick={[Function]}
      >
        Add Venue
      </a>
    </li>
    <li
      className="Nav_logout"
    >
      <a
        href="/login"
        onClick={[Function]}
      >
        Log in
      </a>
    </li>
  </ul>
</nav>
`;
